spec/spec_helper.rb

```diff
@@ -36,4 +36,8 @@
   # the seed, which is printed after each run.
   #     --seed 1234
   config.order = "random"
+
+  config.expect_with :rspec do |c|
+    c.syntax = [:should, :expect]
```

Comment:
I've used it in the past. It's good. I was worried I was making the 'security upgrade' PR too big as it was.. so I left it.